packages/styles/components/_c.tooltip.scss

```diff
@@ -0,0 +1,107 @@
+/* mqp:start */
+.mc-tooltip {
+  $parent: &;
+
+  @include set-font-face();
+
+  position: relative;
+  display: inline-block;
+  margin: 0.25rem;
+
+  &__text {
+    visibility: hidden;
+    background-color: $color-grey-700;
+    color: $color-grey-200;
```

Comment:
I have the impression that this is not an accessible colour that can be used for text. @nherwegh is this voluntary?

Reply:
Nope it's not. It's my mistake I gave `grey-200` for the font-color but it is: `$color-grey-000`